### ISSUE TITLE
training questions

### ISSUE THREAD

**Issue by lmdsx** [dong03/MVSS-Net#9]
Hello!
Thanks for your inspiring work. And I 'm very intersted in it，Among them, there is a little doubt in the training,How is batchsize set during training? After tampering and non-tampering images are shuffled in the dataloader, the loss function used by the images in batch will be different. My training setting, when the batchsize is 8, I don’t know how to calculate the loss，Is it to traverse or set batchsize to 1 or some other method?

**Comment by dong03** [comment]
We return class labels together with images in dataset and send different parts into the corresponding loss function.

e.g.  real part: images[class ==0],  fake part: images[class ==1]

**Comment by hilssen** [comment]
Hello! Can you provide the training code?
